backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java

```diff
@@ -384,7 +383,6 @@ public void listBlobExperimentRuns(
       responseObserver.onNext(response);
       responseObserver.onCompleted();
     } catch (Exception e) {
-      e.printStackTrace();
```

Comment:
if it is not a modeldbexception I would prefer a stack trace , what is the motivation to remove it?